lib/App/Config/Chronicle.pm

```diff
@@ -110,6 +110,29 @@ has setting_name => (
     default  => 'settings1',
 );
 
+=head2 refresh_interval
+
+How much time (in seconds) should pass between C<check_for_update> invocations until
+it actually will do (a bit heavy) lookup for settings in redis.
+
+Default value is 10 seconds
+
+=cut
+
+has refresh_interval => (
+    is       => 'ro',
+    isa      => 'Num',
+    required => 1,
+    default  => 10,
+);
+
+has _updated_at => (
+    is       => 'ro',
+    isa      => 'Num',
+    required => 1,
+    default  => sub { time },
```

Comment:
We don't seem to be updating this, and I'd suggest a default of undef or 0 to ensure that the first time we always pull in the data.